pagopa/io-app
# feat: [#177760591] Added profile name on identification modal

## Short description
Now you can see your name during the identification progress

## How to test
If you are authenticated, try to unlock the app, you will see a label like this "Hi yourName!" on the identification modal

**Before changes:**
<img width="300" alt="Screenshot 2021-04-23 at 10 35 51" src="https://user-images.githubusercontent.com/61834253/115844117-d33f1400-a41f-11eb-863c-dc9f1232b110.png">

**After changes:**
<img width="300" alt="Screenshot 2021-04-23 at 10 32 53" src="https://user-images.githubusercontent.com/61834253/115843738-717eaa00-a41f-11eb-8c7b-2624c11ccf73.png">

For some reason profile name could be undefined, to prevent any error I will show a simple label "Hi!" instead of "Hi profileName!"
